packages/orchestration/src/exos/remote-chain-facade.js

```diff
@@ -94,35 +113,82 @@ const prepareRemoteChainFacadeKit = (
           return asVow(() => {
             const { remoteChainInfo, connectionInfo } = this.state;
             const stakingDenom = remoteChainInfo.stakingTokens?.[0]?.denom;
-            if (!stakingDenom) {
-              throw Fail`chain info lacks staking denom`;
-            }
+            if (!stakingDenom) throw Fail`chain info lacks staking denom`;
+
+            // icqConnection is ultimately retrieved from state, but let's
+            // create a connection if it doesn't exist
+            const icqConnOrUndefinedV =
+              remoteChainInfo.icqEnabled && !this.state.icqConnection
+                ? E(orchestration).provideICQConnection(
+                    connectionInfo.counterparty.connection_id,
+                  )
+                : undefined;
+
+            const makeAccountV = E(orchestration).makeAccount(
+              remoteChainInfo.chainId,
+              connectionInfo.counterparty.connection_id,
+              connectionInfo.id,
+            );
 
             return watch(
-              E(orchestration).makeAccount(
-                remoteChainInfo.chainId,
-                connectionInfo.counterparty.connection_id,
-                connectionInfo.id,
-              ),
-              this.facets.makeAccountWatcher,
+              allVows([makeAccountV, icqConnOrUndefinedV]),
+              this.facets.makeAccountAndProvideQueryConnWatcher,
             );
           });
         },
+        /** @type {ICQConnection['query']} */
+        query(msgs) {
+          return asVow(() => {
+            const {
+              remoteChainInfo: { icqEnabled, chainId },
+              connectionInfo,
+            } = this.state;
+            if (!icqEnabled) {
+              throw Fail`Queries not available for chain ${chainId}`;
+            }
+            // if none exists, make one + still send the query in the handler
```

Comment:
this reads like "make (one + …)". it's only two chars:
```suggestion
            // if none exists, make one and still send the query in the handler
```